src/main/java/cz/cvut/kbss/study/rest/UserController.java

```diff
@@ -101,6 +101,25 @@ public void removeUser(@PathVariable("username") String username) {
         }
     }
 
+    @ConditionalOnProperty(prefix = "security", name = "provider", havingValue = "oidc")
+    @PreAuthorize("hasRole('" + SecurityConstants.ROLE_ADMIN + "') or #username == authentication.name")
+    @PutMapping(value = "/{username}", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    public void updateUserInstitution(@PathVariable("username") String username, @RequestBody User user,
```

Comment:
@kostobog you are updating dedicated controller for "internal" authorization -- see `@ConditionalOnProperty` of this class !!! There is dedicated controller for "oidc" as stated here https://github.com/kbss-cvut/record-manager-ui/issues/12#issuecomment-1846132556 . It should be implemented there.

Reply:
Otherwise it looks ok.

Reply:
Ok, I created a new PR #35 adding updateUserInstitution to OidcUserController.